codecov.yml

```diff
@@ -7,7 +7,7 @@ coverage:
   status:
     patch:
       default:
-        target: 50%
+        target: 0%
```

Comment:
Wondering now if this should be `threshold` ? https://docs.codecov.com/docs/commit-status